targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_KL27Z/device/TOOLCHAIN_ARM_STD/MKL27Z64xxx4.sct

```diff
@@ -51,7 +51,7 @@
 #define __ram_vector_table__            1
 
 #if (defined(__ram_vector_table__))
-  #define __ram_vector_table_size__    0x00000200
```

Comment:
Is this change required? The issue seems to be the large value for the number of NVIC_VECTORS